src/TLuaInterpreter.cpp

```diff
@@ -1141,14 +1179,12 @@ int TLuaInterpreter::getTextFormat(lua_State* L)
     Host& host = getHostFromLua(L);
     QPair<quint8, TChar> result = host.mpConsole->getTextAttributes(windowName);
     if (result.first == 1) {
-        lua_pushnil(L);
-        lua_pushfstring(L, "window \"%s\" not found", windowName.toUtf8().constData());
+        returnWrongArgumentType(L, __func__, QStringLiteral(R"(window "%1" not found)").arg(windowName));
         return 2;
     }
 
     if (result.first == 2) {
-        lua_pushnil(L);
-        lua_pushfstring(L, "current selection invalid in window \"%s\"", windowName.toUtf8().constData());
+        returnWrongArgumentType(L, __func__, QStringLiteral(R"(current selection invalid in window "%1")").arg(windowName));
```

Comment:
Can we agree to just use 'single quotes' inside the string, and do away with R"( syntax?

Reply:
I don't mind the syntax but I'm okay with changing to single quotes

Reply:
Some use R"( currently, some escape \\" every single quote, some do single quotes, some no quotes at all.
It's just a wild mess and rather incoherent. No obvious direction which style to use for new messages.
I would vote for porting \\" and R"( to single quotes, then later add them where no were before.
This way 'argument values' are clearly distinct from parts of the sentence even for casual players.

Reply:
OK

Reply:
TBH I am not a fan of the C++ raw string literal form - partly because it broke when used with texts for translations - though that has now been fixed - and I am *conditioned* to use the escape form - though perhaps I can eventually be deconditioned/deprogrammed away from that. As it is the `'` ***does*** work better within Lua erorr message, it is just that as a C/C++ coder I am also programmed to think of that delimiter only for single `char`s so forgot to use that form.

:exploding_head: